### ISSUE TITLE
Every time I open Powershell now, I get a bunch of questions

### ISSUE THREAD

**Issue by killerkalamari** [dahlbyk/posh-git#199]
I just installed the latest version v4.0-140. Had PS 2.0, and was hoping updating to 3.0 would get rid of all the questions, but it didn't. Screenshot:
![nags](https://cloud.githubusercontent.com/assets/5439104/8097009/74e1a742-0f92-11e5-8473-ed74de2f96c0.png)

**Comment by dahlbyk** [comment]
What does `Get-ExecutionPolicy` show? You likely need to open an elevated prompt and `Set-ExecutionPolicy RemoteSigned`.

**Comment by dahlbyk** [comment]
Or if you downloaded a ZIP of the project from GitHub  you might need to right-click the folder on your Desktop and unblock it.

**Comment by killerkalamari** [comment]
Yes, you're correct I downloaded the ZIP from GitHub. When I right click and go to Properties, I don't see an Unblock option. I had already Set-ExecutionPolicy Unrestricted as a try to fix the error messages (elevated). 

I deleted the folder from my Desktop, and deleted my profile (since PoshGit was the only thing in it). This has resolved the error messages. I guess this also effectively disabled PoshGit too.
